stock_reception_screen_measuring_device/models/stock_reception_screen.py

```diff
@@ -18,6 +18,11 @@ class StockReceptionScreen(models.Model):
         store=True,
         help="Indicates if the package have any measurement missing.",
     )
+    display_package_dimensions = fields.Char(
```

Comment:
```suggestion
    package_dimensions = fields.Char(
```
No need to add a verb here.

Reply:
"display" in "display_name" is not a verb.
Here it's just to say that's dimensions meant to be displayed.
But I can remove it if you still want it to be removed.

Reply:
If it "means to be displayed" then it is a boolean (to display or not something), but if the field contains qualitative information regarding the current record (we are stocking a string here), then `package_dimensions` is enough, no need to add more.
It's my POV, non blocking.